CHANGELOG.md

```diff
@@ -1,5 +1,18 @@
 
 # Changelog
+## 4.3.0 - 2023-02-16
+### Added
+- File attachments for calendar events
+### Changed
+- Lazy load dashboard component
+- Wider input for recurrences
+### Fixed
+- Exception handing for booking controller
+- can_subscribe_link fallback
```

Comment:
i would remove this as looks a bit technical

Reply:
Marked as resolved but not resolved

Reply:
https://github.com/nextcloud/calendar/pull/4988/commits/c1d87cf029515a028ff9358ae5949eb55cb7498b

Reply:
check again

Reply:
wrong line, fixed now